src/App.tsx

```diff
@@ -1,18 +1,118 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useCallback,
+  useContext, useEffect, useRef, useState,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Erorr } from './components/Error/Error';
+import { FilterTodoList } from './components/FilterTodoList/FilterTodoList';
+import { TodoList } from './components/TodoList/TodoList';
+import { NewTodo } from './components/NewTodo/NewTodo';
```

Comment:
Also pay attention for comments from this PR https://github.com/mate-academy/react_todo-app-loading-todos/pull/175#pullrequestreview-1221066307